eng/testing/tests.singlefile.targets

```
@@ -8,6 +8,7 @@
     <UseAppHost>true</UseAppHost>
     <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
     <SelfContained>true</SelfContained>
+    <SingleFileHostSourcePath>$(ArtifactsBinDir)/$(RuntimeIdentifier).$(RuntimeConfiguration)/corehost/singlefilehost</SingleFileHostSourcePath>
```

nit: `ArtifactsBinDir` is already normalized.
```suggestion
    <SingleFileHostSourcePath>$(ArtifactsBinDir)$(RuntimeIdentifier).$(RuntimeConfiguration)/corehost/singlefilehost</SingleFileHostSourcePath>
```